java/server/src/org/openqa/selenium/grid/docker/DockerOptions.java

```diff
@@ -152,7 +153,8 @@ private boolean isEnabled(Docker docker) {
     Image videoImage = getVideoImage(docker);
     loadImages(docker, videoImage.getName());
 
-    int maxContainerCount = Runtime.getRuntime().availableProcessors();
+    int maxContainerCount = Math.min(config.getInt(DOCKER_SECTION, "docker-max-sessions")
```

Comment:
This looks good, but I would prefer to use the same [`--max-sessions`](https://github.com/SeleniumHQ/selenium/blob/9717d403a8fedb6da86c07fac5ccc8f06fb322b3/java/server/src/org/openqa/selenium/grid/node/config/NodeFlags.java#L50) from `NodeFlags`. Like this, the user does not need to think more when configuring this.

Reply:
Thank you! I agree, that makes sense. I will update it.